tests/BenchmarkDotNet.Tests/XUnit/AppveyorWithClassicFrameworkOnlyFactAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace BenchmarkDotNet.Tests.XUnit
+{
+    public class AppVeyorWithClassicFrameworkOnlyFactAttribute : AppVeyorOnlyFactAttribute
+    {
+        private const string Message = "Test requires classic .NET Framework";
+
+        private bool classicDotNet
+        {
+            get
+            {
+#if !CORE
```

Comment:
Please, try to avoid `#if !CORE` constructions outside `BenchmarkDotNet.Portability`. E.g., you can introduce a public `RuntimeInformation.IsCore()` method.